examples/weather/fcn_afno/conf/config.yaml

```diff
@@ -20,12 +20,26 @@ hydra:
   run:
     dir: ./outputs/
 
+train_dir: ./dataset/train
+validation_dir: ./dataset/val
```

Comment:
also, for internal datasets, this will have to be `/data/test/`

Reply:
Fixed.